package.json

```
@@ -0,0 +1,45 @@
+{
+  "name": "bg",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "dev": "next",
+    "build": "next build",
+    "start": "next start",
+    "stylecss": "stylelint './components/**/*.js'"
```

Maybe it'd be better to name this script as `lint:css`

---

Also why are we linting only `components` folder? We have some css in different folders.
```suggestion
    "stylecss": "stylelint './**/*.js'"
```